intermol/lammps/lammps_parser.py

```diff
@@ -1203,9 +1199,13 @@ def write(self, unit_set='real'):
 
             # non-bonded
             if atom_charges:
-                f.write('pair_style lj/cut/coul/cut 9.99999 19.99999\n')  # TODO: match mdp
+                if self.in_file.endswith('_vacuum.input'):
+                    f.write('pair_style lj/cut/coul/long 9.99999 19.99999\n')
```

Comment:
We probably want to make this longer (but I'll fiddle with this).  Will probably involve making the boxes bigger as well.